### PR TITLE
ci: Fix PR title for pre-commit autoupdate PRs

### PR DESCRIPTION
**Pull request by matthewelwell**
## Changes

Fixes PR title (by fixing the initial commit message)

## How did you test this code?

n/a - needs testing after merge and the pre-commit workflow next runs.

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -37,3 +37,6 @@ repos:
     hooks:
       - id: poetry-check
         args: ['-C', 'api']
+
+ci:
+  autoupdate_commit_msg: 'ci: pre-commit autoupdate'
```
Comment (khvn26) [inline .pre-commit-config.yaml:42]: No need to include this in the changelog, IMO:

```suggestion
  autoupdate_commit_msg: 'chore(ci): pre-commit autoupdate'
```

Comment (matthewelwell) [reply .pre-commit-config.yaml:42]: Happy to change this if we want, but `ci` changes _aren't_ included in the changelog.

Comment (matthewelwell) [reply .pre-commit-config.yaml:42]: See [here](https://github.com/Flagsmith/flagsmith/blob/main/release-please-config.json#L33).

**Review by khvn26 — APPROVED** [review]

**Comment by matthewelwell** [comment]
Since this is a trivial config PR that doesn't need the E2E tests to pass, I'm going to go ahead and merge without retrying the (currently consistently failing) E2E tests.

**Pull request merged**